main.cpp

```diff
@@ -0,0 +1,58 @@
+//CodeU: Q1
+//
+//  Created by Julia Semavina on 17.05.2017
+//
+//  problem specification:
+//      Given two strings, write a method to decide if one is a permutation of the other
+//
+//  comments:
+//      register is ignored
+//
+
+#include <iostream>
+#include <map>
+
+
+char lowercase(char a) {
```

Comment:
It's usually better to use a standard library function if there is one, in this case tolower from <ctype.h>